modules/debug.py

```diff
@@ -4,17 +4,20 @@
 from shared import GlobalVariables
 from shared import GlobalConstants
 from shared import ImageDataCore
+from shared import State
 
 def can_call(func):
     @wraps(func)
     def respond_depending_on_state(*args):
         klass = args[0]
         state = klass.CALL_STATUS
-        if (state is DebugState.BASIC) and not _is_verbose(func, klass):
+        global_state = GlobalVariables.STATE
+        if (global_state is State.NOMINAL) or (global_state is State.TEST):
```

Comment:
This should be different. They are the same result but different reasons entirely.